tracer/test/Datadog.Trace.Security.IntegrationTests/AspNetMvc5.cs

```diff
@@ -98,12 +98,13 @@ public AspNetMvc5(IisFixture iisFixture, ITestOutputHelper output, bool classicM
         [Trait("Category", "EndToEnd")]
         [Trait("RunOnWindows", "True")]
         [Trait("LoadFromGAC", "True")]
-        [Fact]
-        public Task TestSecurity()
+        [Theory]
+        [InlineData("/Health/test&[$slice]")]
```

Comment:
Would it be good test with the old url as well, as you've moved the Theory attribute, it should be good to do.

Reply:
fixed in c44656a